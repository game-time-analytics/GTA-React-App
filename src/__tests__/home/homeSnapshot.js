import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from '../../components/home/home';

describe('Home snapshot test', () => {
  it('renders to DOM correctly', () => {
    const component = renderer.create(<BrowserRouter><Home /></BrowserRouter>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
