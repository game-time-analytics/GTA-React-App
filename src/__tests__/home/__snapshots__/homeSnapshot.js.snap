// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot test renders to DOM correctly 1`] = `
Array [
  <span
    id="signin"
  >
    <a
      href="/signin"
      onClick={[Function]}
    >
      Sign In!
    </a>
  </span>,
  <span
    id="signup"
  >
    <a
      href="/signup"
      onClick={[Function]}
    >
      Sign Up!
    </a>
  </span>,
  <img
    alt="GTA logo"
    src="Football-Analytics-Limitations.jpg"
  />,
]
`;
